src/foam/build/Builder.js

```diff
@@ -458,8 +458,14 @@ console.log("Done.");
 
             if ( func == 'UNKNOWN' ) throw new Error("What's the right foam.FOO function for a " + s.cls_.id);
 
-
-            data += `foam.${func}(${serializer.stringify(filterAxiomsByFlags(s))});\n`;
+            // Print FOAM model definition
+            dataTmp = `foam.${func}(${serializer.stringify(filterAxiomsByFlags(s))});\n`
+            // Print sourceURL annotation for browser inpector
+            if ( typeof s.source !== 'undefined' ) {
+              dataTmp += `\n//# sourceURL=/`+s.source;
+            }
+            // NB: eval() is necessary for `//# sourceURL` to work
```

Comment:
What does NB: mean?

Reply:
"Note well" - I'll remove that from the comment so it's less confusing